### ISSUE TITLE
Ensure supported census strategies allow weighted census

### ISSUE THREAD

**Issue by elboletaire** [vocdoni/ui-scaffold#551]
Addresses census does not allow yet to define weights, and we should have it. Check any other possible census strategies that should have a way to define weight. We should also consider adding a "weighted" census to the census3 manager, if that's even an option.

**Comment by elboletaire** [comment]
Kinda related to https://github.com/vocdoni/ui-scaffold/issues/532, but this should be done before to ensure we have enough support for weighted census.

**Comment by jpaulet** [comment]
Already done.

**Comment by jpaulet** [comment]
Already done.
